Source/JavaScriptCore/heap/GCMemoryOperations.h

```diff
@@ -251,7 +251,7 @@ ALWAYS_INLINE void gcSafeZeroMemory(T* dst, size_t bytes)
     static_assert(sizeof(T) == sizeof(JSValue));
     RELEASE_ASSERT(bytes % 8 == 0);
 #if USE(JSVALUE64)
-#if COMPILER(GCC_COMPATIBLE) && (CPU(X86_64) || CPU(ARM64))
+#if CPU(X86_64) || CPU(ARM64)
```

Comment:
I think we can actually remove this `#if`, so that there's just `if CPU(X86_64)`, `#elif CPU(ARM64)`, and `#else`.

Reply:
Done